src/mixins/test/__snapshots__/triangle.test.js.snap

```diff
@@ -83,7 +91,8 @@ Object {
 
 exports[`triangle should properly render bottomRight pointing arrow with blue foregroundColor, width of 20px and height 20px 1`] = `
 Object {
-  "borderColor": "transparent transparent blue transparent",
+  "borderBottomColor": "blue",
+  "borderColor": "transparent",
```

Comment:
Unfortunately, key order matters a lot for this particular use case. `borderColor` will override `borderBottomColor` and make the element transparent. We had to do some magic in the original implementation to account for this: https://github.com/styled-components/polished/blob/8607d641e27d945612e81a3078b1674200d6286e/src/mixins/triangle.js

Reply:
Is the object always sorted by key or is it just an artifact of the testing framework?

Did you mean using the border color helper or adding `!important` to borderBottomColor?